arraysum.c

```diff
@@ -0,0 +1,25 @@
+#include <stdio.h>
+
+int main(){
```

Comment:
Nice job, but you got the assignment instructions mixed up! You weren't supposed to do the cumulative sum, you were supposed to do the sum of all assignments BUT the specific element in that spot. Be careful next time! Good commenting though